queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html

```diff
@@ -0,0 +1,33 @@
+Attention **{{ account_name }}**.
+
+Your BC Registries and Online Services account is listed as the account managing **{{ business_name }}**.
+
+The following account is requesting authorisation to manage Registries activities for {{ business_name }}:
+
+**{{ requesting_account }}**
+
+Only authorise this account if you recognize it.
+
+Allowing **{{ requesting_account }}** to manage all Registries activities for **{{ business_name }}**, will allow it to do the following (including but not limited to):
+* Authorisation of accounts to manage the business,
+* Appointment and cessation of directors,
+* Dissolution of the business,
+* Filing of Annual Reports, and
+* Changing the records office of the business.
+
+{% if additional_message %}
+**Message from the Requesting Account:**
+
+{{ additional_message }}
+{% endif %}
+
+**Note:** This request has been sent to all Account Administrators and Coordinators of {{ to_acount_name }}.
+
+To manage this request from **{{ from_account_name }}** to manage Registries activities for {{ to_acount_name }} follow the link below:
+
+To authorise or not authorise **{{ from_account_name }}** to manage Registries activities for **{{ business_name }}**, log in to your BC Registries Account:
+
+
+If you have questions about this request please contact the Business Registry:
+[Business Registry contact component]
```

Comment:
Can someone suggest how to add this ?
I tried to figure it out, and am not sure why, when sending email to my Gmail account I dont see these, only Header I think.

Reply:
This looks like something from footer you need?

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/d41c7ecb-0633-4279-900c-1e7d4c08813c)

Reply:
Just put it right in the template like this one:

![image](https://github.com/bcgov/sbc-auth/assets/3484109/6e318ba6-0625-4f0c-8b99-355952ea0e50)